dotnet/sample-apps/aws-sdk/wrapper/SampleApps/SampleApps/Function.cs

```diff
@@ -0,0 +1,48 @@
+using Amazon.Lambda.APIGatewayEvents;
+using Amazon.Lambda.Core;
+using Amazon.S3;
+using OpenTelemetry;
+using OpenTelemetry.Contrib.Instrumentation.AWSLambda.Implementation;
+using OpenTelemetry.Trace;
+using System;
+
+// Assembly attribute to enable the Lambda function's JSON input to be converted into a .NET class.
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.Json.JsonSerializer))]
+
+namespace SampleApps
```

Comment:
Can you call this `AwsSdkSample` or similar? Currently it's `SampleApps/SampleApps`, but I think each app is supposed to have a name

Reply:
Just updated, and it turns out there needs to be a space between each item in terraform config